docs/instruction.md

```diff
@@ -82,7 +82,7 @@ where "XXX" can be "wal" for the Wallis filter, "hps" for the trivial high-pass
        MadScalar                   Scalar to be multiplied by Mad used as threshold for disparity filtering of the chip displacement deviation from the median (default = 4)
        
        ------------------parameter list: miscellaneous------------------
-       WallisFilterWidth           Width of the filter to be used for the preprocessing (default = 21)
+       WallisFilterWidth           Width of the filter to be used for the preprocessing (default = 5)
```

Comment:
Looks like the default filter width was set to 21 in autoRIFT -- I've changed it here, and these two places: [`autoRIFT.py`](https://github.com/nasa-jpl/autoRIFT/pull/59/files#diff-aef32708be8cb7db896244136a3694fcc729dc6cd5f03753863b2387c579dae1R646), and [`autoRIFT_ISCE.py`](https://github.com/nasa-jpl/autoRIFT/pull/59/files#diff-b2c5acdf353417a485d8b31895494bc204ed04119c199d85ca3bcd256a7fccefR38).

Unfortunately, it looks like right now, the default value was used for all missions as it's used by `preprocess_filt_hps()`:
https://github.com/nasa-jpl/autoRIFT/pull/59/files#diff-aef32708be8cb7db896244136a3694fcc729dc6cd5f03753863b2387c579dae1L118


which was always applied:
https://github.com/nasa-jpl/autoRIFT/pull/59/files#diff-0da0f6848bb25f16593e48fa47fea8248d3dd9214f85b82e6a007381d8a1ffdaL285

This *probably* should have been adjusted on a per-mission basis to keep the filterwidth to the same ground resolution.

Reply:
I'll generate some output products for all missions with this new setting and we can decide how we want to handle the filterwidth